src/config/Config.zig

```diff
@@ -609,10 +609,8 @@ palette: Palette = .{},
 /// than 0.01 or greater than 10,000 will be clamped to the nearest valid
 /// value.
 ///
-/// A value of "1" (default) scrolls the default amount. A value of "2" scrolls
-/// double the default amount. A value of "0.5" scrolls half the default amount.
-/// Et cetera.
-@"mouse-scroll-multiplier": f64 = 1.0,
+/// A value of "3" (default) scrolls 3 lines per tick.
+@"mouse-scroll-multiplier": f64 = 3.0,
```

Comment:
I think we could change this to a usize, or even a u8. Even the original comment says this only applies to wheel mice, so changing the meaning to be "lines per tick" makes sense to me - and this should only ever be an integer.

Reply:
Since the default is now `3` I think that changing to an integer make sense, you're still able to slow it down if you want to.

Reply:
I think the impact of this multiplier is meant to affect precision scrolling as well, which comes in at fractional deltas. So I think this should be float. We can continue this discussion though no need to block the PR on it. I know this PR removes that behavior but I'm not sure if we want that to stick or not, I recall during the beta some people wanting to change the momentum scrolling on macOS to do more or less.